README-code-reviews.md

```diff
@@ -0,0 +1,103 @@
+Code Review Policies
+====================
+
+Project Stewardship
+-------------------
+
+The OpenJFX Project is guided by the Project Leads and Reviewers.
+
+__Project Co-Lead__: Kevin Rushforth (kcr) <br>
+__Project Co-Lead__: Johan Vos (jvos)
```

Comment:
There are two sets of ids - one for OpenJFX/JBS and one for Github.  This might be confusing sometimes, should we list both?

Reply:
Not sure what you mean, but it doesn't seem related to this PR.

Reply:
He means people reading reviews are usually on github and your github ID is kevinrusforth,
so they might be confused by "kcr" which is your openjdk id.

Since the context is identifying OpenJDK Project leads, I think the OpenJDK ID is the right and only one to include here. At most you could clarify  like (OpenJDK ID - kcr)

Reply:
exactly my point.  when the majority of time is spent in the context of github (PR review etc) it might be confusing when one types `@jvos` and it's not found, or when within a PR there are references to both `jvos` and `johanvos`

(see the very first comment in https://github.com/openjdk/jfx/pull/1388 )

Reply:
fixed